.ibm/pipelines/value_files/values_showcase.yaml

```diff
@@ -52,10 +52,10 @@ global:
           kubernetes:
             clusterLocatorMethods:
               - clusters:
-                - authProvider: serviceAccount
-                  name: 'my-cluster'
-                  serviceAccountToken: ${K8S_CLUSTER_TOKEN_ENCODED}
-                  url: ${K8S_CLUSTER_API_SERVER_URL}
+                  - authProvider: serviceAccount
```

Comment:
Were these extra spaces actually needed?  There's a few changes in these files which seem more like a different tab width setting.  Not necessarily a big deal (except YAML) but does create extra unnecessary changes.